src/victory-group-container/victory-group-container.js

```diff
@@ -0,0 +1,113 @@
+import React, { PropTypes } from "react";
+import { ClipPath } from "../victory-primitives/index";
+
+export default class VictoryGroupContainer extends React.Component {
```

Comment:
this is kind of a confusing name since it has no relation to either `VictoryGroup` or `VictoryContainer`, no?